README.md

```diff
@@ -36,6 +36,14 @@ pip uninstall qiskit_code_assistant_jupyterlab
 
 For help setting up and using the Qiskit Code Assistant read [GETTING_STARTED](GETTING_STARTED.md).
 
+### Terms of use
```

Comment:
Are you sure this is the best place for this section? I think moving it to the bottom and making it a H2 (`##`) instead would be closer to where users expect it

Reply:
Done, thanks https://github.com/Qiskit/qiskit-code-assistant-jupyterlab/pull/4/commits/8fb827532a124712f9bfda0e54603322c0f5f667